svo (reboundingsileristracker).xml

```diff
@@ -236,7 +236,7 @@ svo.rt_razed = nil</script>
                 <colorTriggerFgColor>#000000</colorTriggerFgColor>
                 <colorTriggerBgColor>#000000</colorTriggerBgColor>
                 <regexCodeList>
-                    <string>^\w+ takes a long drag off (?:his|her) ?.*pipe\.$</string>
+                    <string>^\w+ takes a long drag off (?:his|her)? ?.*pipe\.$|^A great gurgling sound is heard as \w+ takes a long drag from an enamel bubble pipe\. A series of shining bubbles erupt from the bowl of (?:her|his) pipe\.$</string>
```

Comment:
Didn't notice the pipe character. Why not make another pattern in the same trigger though? It would be more readable and be less concerning on performance.